packages/core/src/Stores/portfolio-store.js

```diff
@@ -579,4 +580,8 @@ export default class PortfolioStore extends BaseStore {
     get is_multiplier() {
         return this.contract_type === 'multiplier';
     }
+
+    get is_turbos() {
```

Comment:
@maryia-deriv Please add the type in `packages/stores/types.ts` 🙏🏻

Reply:
@farzin-deriv unfortunately, the scope of the current PR doesn't include TS migration of portfolio-store, and there are no types for portfolio-store in `packages/stores/types.ts` yet.
<img width="776" alt="Screenshot 2023-03-28 at 5 02 11 PM" src="https://user-images.githubusercontent.com/103177211/228263180-276a0a51-d105-43b8-bdfe-cf173fd9cd11.png">